src/Spryker/Zed/UuidBehavior/Persistence/Propel/Behavior/templates/objectUpdateUuidAfterInsert.php

```diff
@@ -0,0 +1,13 @@
+
+/**
+ * @param ConnectionInterface $con
+ *
+ * @return void
+ */
+protected function updateUuidAfterInsert(ConnectionInterface $con = null)
+{
+    if (empty($this->getUuid())) {
```

Comment:
Please do not use empty with methods. Use just

    if (!$this->getUuid()) 

etc